oio/common/http.py

```diff
@@ -258,7 +259,8 @@ def pop(self, k, default=None):
 
 def get_pool_manager(pool_connections=DEFAULT_POOLSIZE,
                      pool_maxsize=DEFAULT_POOLSIZE,
-                     max_retries=DEFAULT_RETRIES):
+                     max_retries=DEFAULT_RETRIES,
+                     backoff_factor=DEFAULT_BACKOFF):
```

Comment:
Don't forget to add `'backoff_factor'` [here](https://github.com/open-io/oio-sds/blob/4.1.12/oio/api/base.py#L26).